docs/tests/conditionals.rst

```diff
@@ -0,0 +1,188 @@
+a basic example.
+
+.. code:: yaml
+
+    basic_test:
+        only_if:
+            user: ['calvin']
+        run:
+            cmds:
+                ...
+
+    In this example the dictionary following 'only_if' has a key 'user'
+    and the value of 'calvin'. Pavilion will resolve the key and
+    check who the actual user is. Pavilion will then compare the resolved
+    key with the value and if 'user == calvin' then the test will run
+    as 'calvin == calvin'.
+
+Conditional statements can also parse dictionaries with multiple keys
+and values, let's look at a more complicated test.
+
+.. code:: yaml
+
+    advanced_test:
+        only_if:
+            user: ['calvin', 'paul']
+            weekday: ['Monday', 'Wednesday', 'Friday']
+        run:
+            cmds:
+                ...
+
+    In this example we have two keys that need to be resolved, user and
+    weekday. As with the simpler example above, each key is
+    resolved and checked with the values supplied to it. An important
+    thing to remember is 'only_if' needs to have a match on ALL keys.
+    If the user is paul but it's Tuesday,the test will not run.
+
+not_if
+~~~~~~
+
+The ``not_if`` section of the pavilion test config behaves exactly
+like the ``only_if`` section except for one main difference. The
+test will be assigned the status of ``SKIPPED`` if at least one
+conditional match is found. Let's take a look at an example:
+
+.. code:: yaml
+
+    basic_test:
+        not_if:
+            sys_os: ['windows']
+        run:
+            cmds:
+                ...
+
+    In this example the 'not_if' directive is followed by a dictionary
+    with the key 'sys_os' and the value of 'windows'. Pavilion will resolve
+    the key and check what the operating system on the machine is. If
+    sys_os resolves to 'windows' then the 'not_if' conditions has a match
+    and the test will be skipped.
+
+Conditional statements can also parse dictionaries with multiple keys
+and values, let's look at a more complicated test.
+
+.. code:: yaml
+
+    advanced_test:
+        not_if:
+            user: ['calvin', 'nick']
+            sys_os: ['linux']
+        run:
+           cmds:
+               ...
+
+    In this example Pavilion will need to resolve two keys, user,
+    sys_os. When the keys are resolved they will be compared
+    to the dictionary values supplied to them. It is important to note
+    for 'not_if', it only takes 1 match to skip the test. In this case
+    if the user is either 'calvin' or 'nick', or the operating system
+    is 'linux', the test will be skip.
+
+Variables
+~~~~~~~~~
+
+Throughout this documentation variables are synonymous with keys. Keys
+being the literal dictionary key supplied after calling ``not_if`` or
+``only_if``. There are multiple types of variables supported in Pavilion
+and for detailed documentation on what variables to use, and how to create
+you own variables see `Variables <variables.html>`__.
```

Comment:
We're going to change this (to '{{user}}'. You will need to note somewhere that having variables in keys is _unique_ to these sections of the Pavilion test configs.